src/trade/tradeConf.es6

```diff
@@ -163,8 +163,11 @@ const register_ticks = (state, extra) => {
    fn = liveapi.events.on('proposal_open_contract', (data) => {
       const contract = data.proposal_open_contract;
       if(contract.contract_id !== extra.contract_id) return;
-      entry = contract.entry_tick_time * 1;
-      expiry = contract.date_expiry * 1; /* date_expiry gets updated when contract is settled !!! */
+      entry = contract.entry_tick_time ? contract.entry_tick_time * 1 : entry;
```

Comment:
@arnabk this is the same change I submitted for master. Includes amin's changes too in the following line. I added these because I required them for testing my changes in chartingRequestMap.es6.